smartdashboard/Experiment_Overview.py

```diff
@@ -52,48 +52,17 @@ def build_app(manifest_path: str) -> None:
 
     try:
         manifest = load_manifest(manifest_path)
+        st.session_state["manifest"] = manifest
```

Comment:
I'm not actually trying to access it in `Experiment_Overview.py`, so I left it out because I think forcing the manifest to be reloaded when going back to this page isn't a bad idea. However, now that the manifest file is tracked, maybe it should be added in so it's not reloaded when unnecessary? I'm a little torn.